### ISSUE TITLE
KVM/Virtualization Detection

### ISSUE THREAD

**Issue by glasschef** [chef/ohai#621]
On a "specific" KVM instance /proc/cpuinfo contains: 
model name : Intel Core i7 9xx (Nehalem Class Core i7) 
and that obviously fails KVM detection.

Maybe we can add in something like this to linux virtualization detection?

``` 
virtualization[:system] = "openstack" 
virtualization[:role] = "guest" 
virtualization[:systems][:openstack] = "guest" 
when /Manufacturer: Red Hat/ 
   if so.stdout =~ /Product Name: KVM/ 
      virtualization[:system] = "kvm" 
      virtualization[:role] = "guest" 
      virtualization[:systems][:kvm] = "guest" 
   end 
   else
      nil 
end
```

**Comment by pburkholder** [comment]
@mcquin Hi Claire -- is this something you think you can pull in to Ohai? Thanks, Peter

P.S. This internally is also https://getchef.zendesk.com/agent/tickets/5902 -- so a customer request.

**Comment by mcquin** [comment]
@glasschef Which version of Ohai are you using?

[This commit](https://github.com/chef/ohai/commit/444ae0f793ed380756164657ae28222bc2ffaf83) should address the above issue. It is available with Ohai 8.6.0.

**Comment by btm** [comment]
I just checked the ZD issue and 8.6.0 solved the bug for the user.
